app/views/posts/index.html.erb

```diff
@@ -1,8 +1,24 @@
-<h1 style='text-align:center'>Post list</h1>
+<div style="margin-top: 10px">
+ <% if @user_search.nil? %>
+<h4>User not found</h4>
+<% else %>
+<div>
+<%= render partial: 'users/user', locals: { user: @user_search } %>
+<div>
+<% end %>
+</div>
+<div>
 <%= @posts.each_with_index do |post, index| %>
```

Comment:
- [ ] When looping, you don't need the `erb` tags that render. You just need the normal `erb` tags. If you use the `erb` tags that render, you will display the result of the query as seen in the image below:

![image](https://user-images.githubusercontent.com/77106962/225450393-7f80376f-fa0f-4116-9772-819afd6c8d62.png)

Kindly adjust it to `<% @posts.each_with_index do |post| %>`